README.md

```diff
@@ -1,43 +1,30 @@
-# The One and Only
+# merge-conflict-demo
 
-I am the one and only, oh yeah
+Matt is the one and only, oh yeah
 Call me, call me by my name or call me by number
 You put me through it
 I'll still be doing it the way I do it
 And yet, you try to make me forget
 Who I really am, don't tell me I know best
 I'm not the same as all the rest
-
-I am the one and only
-No place I'd rather be
-I am the one and only
-You can't take that away from me
-
+Nobody I'd rather be
+Matt is the one and only
+You can't take that away from him
 I've been a player in the crowd scene
-A dot on the big screen
+A flicker on the big screen
 My soul embraces one more in a million faces
 High hopes and aspirations, and years above my station
 Maybe but all this time I've tried to walk with dignity and pride
-
-We are the two and together
+Matt is the one and only
 Nobody I'd rather be
-I am the one and only
-You can't take that away from me
-
+Matt is the one and only
+You can't take that away from him
 I can't wear this uniform without some compromises
+But I can wear a bib (unlike Mario)
 Because you'll find out that we come
-In different shapes and sizes
-No one can be myself like I can
-For this job I'm the best man
-And while this may be true
-You are the one and only you
-
-I am the one and only
-Nobody I'd rather be
-I am the one and only
-You can't take that away from me
 
 I am the one and only
 Nobody I'd rather be
 I am the one and only
 You can't take that away from us
+>>>>>>> main
```

Comment:
This looks like a merge marker that has been left in?

Reply:
ok